script.js

```diff
@@ -0,0 +1,65 @@
+// Elements
+const quoteContainer = document.getElementById('quote-container')
+const quoteTextEl = document.getElementById('quote')
+const quoteAuthorEl = document.getElementById('author')
+const twitterBtn = document.getElementById('twitter')
+const newQuoteBtn = document.getElementById('new-quote')
+const loader = document.getElementById('loader')
+
+loader.hidden = true
```

Comment:
if this is only used inside loadingShowOrHide, then it would be better to have it inside that scope

Reply:
I can put "newQuoteBtn" inside of the function but I can't think of a way to put "loader." Because I have to set it's "hidden" property to be "true" once and then it changes every time the fn gets called.

Reply:
loader.hidden is false by default, so no need to set it

Reply:
that's why I'm setting loader.hidden to true at first so, the container and the loader don't show on the screen at the same time. and the function just changes to the opposite of what they are(hidden property)